packages/environment-check/src/output/zip.ts

```diff
@@ -25,8 +25,13 @@ function byteNumberToSizeString(byteNumber: number): string {
  *
  * @param results - environment check results
  * @param targetFile - path and filename of target zip archive. Default is 'envcheck-results.zip'.
+ * @param markdownOnly - only include markdown in the zip
  */
-export function storeResultsZip(results: EnvironmentCheckResult, targetFile = 'envcheck-results.zip'): void {
+export function storeResultsZip(
+    results: EnvironmentCheckResult,
+    targetFile = 'envcheck-results.zip',
+    markdownOnly = false
```

Comment:
In which case do we need `markdownOnly`? I think if we store check results as zip it is always better to include json and markdown. @theStiggySpd was there a specific request?

Reply:
I did raise this in grooming and it was requested to provide markdown only.